### DISCUSSION TITLE
FR: Monitor Wraparound support

### DISCUSSION THREAD

**Discussion by JamieKM** [rxhanson/RectanglePro-Community#259] [Ideas]
I have an idea for a feature I call Wraparound, which comes from an old piece of software of the same name that's no longer supported. The idea is that you could select sides of a monitor that, when enabled, allows the cursor to wrap around to the other edge. For example, in a 2 monitor setup, when your cursor moves to the left edge of the left monitor, instead of stopping, it wraps around to the right edge of the right monitor. Or if your cursor moves to the top of your monitor, instead of stopping, it wraps around to the bottom of the monitor, basically eliminating monitor edges. On large, 2-monitor systems, it's a godsend. Here's an image of the product's Preferences, indicating wraparound edges on the left edge of the left monitor and right edge of the right monitor:

<img width="607" alt="166104460-38ec2fa2-afa8-4b49-8d0e-7894c2e626ed" src="https://user-images.githubusercontent.com/104679784/232756596-f1b4ed29-aa4f-4f69-af20-61cec2e18964.png">

Thoughts?

**Comment by rxhanson** [comment]
Thanks for the idea. I am planning on a release dedicated to cursor movement, so this would fit in there. It's probably a few releases out at this point. Seems like at a minimum there could just a be a checkbox in the preferences for it, where the logic would be, if there's two+ displays in a row, wrap the cursor. Beyond that, there can be a cursor speed specification for how fast it must move to wrap, and/or a modifier key recorder so you could make it wrap only while holding a modifier key. Am I missing anything?

**Reply by JamieKM** [reply]
Ahh, that's good to hear. As for the original app, it had an option to run in only certain applications. And I wouldn't think you'd need to check for 2+ displays...I imagine it would be useful for people with these giant 32"+ displays as well. But I do like the modifier key idea.

**Reply by JamieKM** [reply]
Hey Ryan, circling back to this request...just curious if there was any traction on this?

**Reply by rxhanson** [reply]
Thanks for checking in! Of course, I never did get around to the cursor movement release that I would have added this in with. I still would like to do this, but I'm not sure when it would make it in.
